### PR TITLE
Added new HWS, CT and HX canonical types, new riser general and abstracts

### PR DESCRIPTION
**Pull request by evgeniya-engel**

### CODE REVIEW

**Comment by evgeniya-engel** [comment]
@cstirdivant hey Claire, this one is ready for review. Couldn't add you as a reviewer.

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 1]
```diff
@@ -1819,6 +1822,12 @@ PWDT:
   uses:
   - primary_return_water_temperature_sensor
   - primary_supply_water_temperature_sensor
+  opt_uses:
+  - primary_supply_water_heating_thermal_power_sensor
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:1826]: remove `supply`

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 2]
```diff
@@ -5611,6 +5638,43 @@ SWTCSCM:
   - supervisor_alarm
 
 
+SESWTC:
+  guid: "ac0191bf-f1ba-4dd9-ba81-d31515589c2c"
+  description: "Secondary loop supply water temperature control."
+  is_abstract: true
+  implements:
+  - OPERATIONAL
+  opt_uses:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:5647]: suggest only adding opt uses that are in your instance

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/HX.yaml [hunk 3]
```diff
@@ -206,4 +206,15 @@ HX_CHWDT_PRWDT_CHWDPM_PWDPM:
   - CHWDT
   - PRWDT
   - CHWDPM
-  - PWDPM
\ No newline at end of file
+  - PWDPM
+
+HX_CWRISOVPM_CHWRWDVPM_CWDT_CHWDT:
+  guid: "993e124b-b666-46ee-95ee-e958423ada9e"
+  description: "Heat exchanger with condending return isolation valve, chilled return diverting valve, and chilled and condensing temperature differential monitoring."
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/HX.yaml:213]: typo in this description

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**